app/bundles/CoreBundle/Assets/js/1.core.js

```diff
@@ -895,3 +895,33 @@ var Mautic = {
         }
     }
 };
+
+/**
+ * Allow us to have disabled state for tabs
+ */
+function toggleLinkAttributes() {
```

Comment:
Recurring issue, as seen in #13948 and #14043 : 

- functions need to be nested within the `Mautic` namespace.
- In this case, since it's tab-related code, it should probably live in `12.tabs.js`

Reply:
Nice catch, thanks! 🙏🏻